template-parts/home-page-banner.php

```diff
@@ -0,0 +1,21 @@
+<?php if( get_field("enable_home_page_title") && get_field("home_page_title")
+          && get_field("home_page_image") ): ?>
```

Comment:
If you change the custom field to require the title and the image then you don't need the last 2 cases.